examples/plotting/multi_scale.py

```diff
@@ -1,3 +1,13 @@
+'''This example shows a graph with multiple scales on y-axis.
+This can be used to depict values in two different units.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.line, bokeh.plotting.figure.circle,
+    bokeh.plotting.figure.add_layout
+    :refs: :ref:`userguide_plotting_figure`
+    :keywords: multiple, scale, graph
```

Comment:
I don't think "multiple" is a good keyword here, that could refer to nearly anything

Reply:
Ok, will remove those.